bfmisc

```diff
@@ -33,22 +33,6 @@ set -e
 # This script executes miscellaneous tasks. These are vendor/customer
 # specific operations that are required during installation procedure.
 ##
```

Comment:
Q: does it make sense to keep this script? I think the bcfg approach gives us more flexibility than this script, right?

Reply:
Done. Removed